### PR TITLE
New package: paperwork-2.2.2

### PR DESCRIPTION
**Pull request by idealist1508**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by abenson** [comment]
Each package should have its own commit, following [these guidelines](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes).

File: srcpkgs/libinsane/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+version=1.0.10
+revision=1
+build_style=gnu-makefile
+hostmakedepends="meson pkg-config sane-devel vala-devel glib-devel gobject-introspection gtk-doc"
+depends="sane"
+short_desc="Crossplatform library to access scanners"
+maintainer="idealist1508 <id1508-github@yahoo.com>"
+license="LGPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/OpenPaperwork/libinsane"
+#changelog=""
```
Comment (abenson) [inline srcpkgs/libinsane/template:12]: Remove unused variables.

File: srcpkgs/libinsane/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'libinsane'
+pkgname=libinsane
+version=1.0.10
+revision=1
+build_style=gnu-makefile
+hostmakedepends="meson pkg-config sane-devel vala-devel glib-devel gobject-introspection gtk-doc"
```
Comment (abenson) [inline srcpkgs/libinsane/template:6]: Most of the `-devel` packages should go in `makedepends`

File: srcpkgs/python3-fabulous/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'python3-fabulous'
+_pkgname=fabulous
```
Comment (abenson) [inline srcpkgs/python3-fabulous/template:2]: No need for this, `${pkgname//python3-/}` is enough

File: srcpkgs/python3-fabulous/template [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+pkgname=python3-fabulous
+version=0.4.0
+revision=1
+build_style=python3-module
+makedepends="python3-build python3-installer python3-setuptools_scm python3-wheel"
+short_desc="Print images, colors, and stylish text to the terminal with Python"
+maintainer="idealist1508 <id1508-github@yahoo.com>"
+license="Apache-2.0"
+homepage="https://github.com/jart/fabulous"
+distfiles="https://github.com/jart/$_pkgname/releases/download/$version/$_pkgname-$version.tar.gz"
```
Comment (abenson) [inline srcpkgs/python3-fabulous/template:12]: Here, just use `$version`, no need for the `$_pkgname`

File: srcpkgs/python3-pillowfight/template [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-pillowfight'
+pkgname=python3-pillowfight
+version=0.3.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3 python3-wheel python3-pip"
```
Comment (abenson) [inline srcpkgs/python3-pillowfight/template:6]: Why does it need `pip`?

**Comment by idealist1508** [comment]
> Each package should have its own commit, following [these guidelines](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes).

Done

File: srcpkgs/python3-pillowfight/template [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+depends="python3-pillow-simd"
+checkdepends="python3-pillow-simd"
+short_desc="Python bindings for libpillowfigh"
+maintainer="idealist1508 <id1508-github@yahoo.com>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/OpenPaperwork/libpillowfight"
+python_version=3
+
+_COMMIT="50d965879eb89fdef9be09d6e934329486ff585d"
+distfiles="https://gitlab.gnome.org/World/OpenPaperwork/libpillowfight/-/archive/${_COMMIT}/libpillowfight-${version}.tar.bz2"
```
Comment (abenson) [inline srcpkgs/python3-pillowfight/template:17]: Why not `https://gitlab.gnome.org/World/OpenPaperwork/libpillowfight/-/archive/${version}/libpillowfight-${version}.tar.gz`

Comment (abenson) [reply srcpkgs/python3-pillowfight/template:17]: Same for the others.

Comment (idealist1508) [reply srcpkgs/python3-pillowfight/template]: I used Arch PKGBUILD as a template.

File: srcpkgs/python3-pyocr/template [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+depends="tesseract-ocr"
+short_desc="Python wrapper for Tesseract and Cuneiform"
+maintainer="idealist1508 <id1508-github@yahoo.com>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/OpenPaperwork/pyocr"
+distfiles="https://gitlab.gnome.org/World/OpenPaperwork/pyocr/-/archive/${version}/pyocr-${version}.tar.bz2"
+checksum=ba9c63cfac7bb6e441590053954466b41768e6b87f1b279c2d62460cad8bb174
+
+do_build() {
+	python -m build --wheel --no-isolation
```
Comment (idealist1508) [inline srcpkgs/python3-pyocr/template:16]: I have no idea why this line causes a build error on any architecture except x86_64 glibc and i686 glibc.

`/void-packages/srcpkgs/python3-pyocr/template: line 16: python: command not found
=> ERROR: python3-pyocr-0.8.5_1: do_build: 'python -m build --wheel --no-isolation' exited with 127
=> ERROR:   in do_build() at srcpkgs/python3-pyocr/template:16
`

File: srcpkgs/python3-pyocr/template [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-pyocr'
+pkgname=python3-pyocr
+version=0.8.5
+revision=1
+build_style=python3-module
```
Comment (abenson) [inline srcpkgs/python3-pyocr/template:5]:
```suggestion
build_style=python3-pep517
makedepends="python3-poetry-core"
```

File: srcpkgs/python3-pyocr/template [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+makedepends="python3-build python3-installer python3-setuptools_scm python3-wheel"
```
Comment (abenson) [inline srcpkgs/python3-pyocr/template:6]:
```suggestion
makedepends="python3-poetry-core"
```

File: srcpkgs/python3-pyocr/template (hunk 7 above)
Comment (idealist1508) [reply srcpkgs/python3-pyocr/template:16]: I found a solution.

**Pull request closed without merging**